package.json

```diff
@@ -1,8 +1,12 @@
 {
   "scripts": {
+    "i18nExport": "bundle exec rake i18n:js:export",
     "test": "jest"
   },
   "jest": {
+    "roots": [
+      "<rootDir>/spec/javascript"
```

Comment:
This lets Jest check for a `__mocks__` directory under `spec/javascript` in order to mock node_modules outside of `setupTests.js`. This is quite common ([docs](https://jestjs.io/docs/en/manual-mocks)).